### ISSUE TITLE
Alternative text of images should not be repeated as text

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#4447]
## Area or Component
Static Site sub pages
https://www.simplereport.gov/getting-started/
https://www.simplereport.gov/support/
https://www.simplereport.gov/using-simplereport/
https://www.simplereport.gov/user-guide/

## Quick Summary
If you inspect the simple report logo on the pages above you will notice it has a `usa-sr-only` `<h1>` for the logo and a `<h1> `with the actual title of the page. The `<h1>` for the logo also repeats what is being said in the img's alt text.

### How to Replicate
See above

## Accessibility Violation
https://dequeuniversity.com/rules/axe/4.4/image-redundant-alt?application=AxeChrome

## Action Requested
Remove the `<h1>` on those pages since they repeat what is in the simple report logo alt text.

## Screenshots (optional)
Example /support page
![Screen Shot 2022-10-11 at 19.59.16.png](https://images.zenhubusercontent.com/5ffc8b9b4950f50bf5829d88/1b3ad412-74fb-47a7-9cc2-0446c631a6f7)

![Screen Shot 2022-10-11 at 19.59.12.png](https://images.zenhubusercontent.com/5ffc8b9b4950f50bf5829d88/8a458d06-e32a-407e-bd46-d2214fed75e1)

![Screen Shot 2022-10-11 at 19.59.23.png](https://images.zenhubusercontent.com/5ffc8b9b4950f50bf5829d88/24063b88-29a5-42fc-a2a4-bcab9d0cc67d)

## Source Code (optional)

[which file or line is the issue attached to?]: #
